mods/ctf/ctf_anticombatlog/mod.conf

```
@@ -0,0 +1,2 @@
+name = ctf_anticombatlog
+depends = ctf,dropondie,default
```

Better understanding.
```suggestion
depends = ctf, dropondie, default
```

---

I don't think spaces are allowed here.

---

They're allowed, and it works.

---

I prefer to not add spaces, sorry.